### PR TITLE
Don't skip if branch contains "release"

### PR DESCRIPTION
**Pull request by chuckwagoncomputing**
#3817

### CODE REVIEW

File: .github/workflows/build-firmware.yaml [hunk 1]
```diff
@@ -194,9 +194,9 @@ jobs:
 
     steps:
     - name: Execution throttle early exit
-      if: ${{ matrix.skip-rate }}
+      if: ${{ matrix.skip-rate && !contains(github.ref, 'release') }}
```
Comment (rusefillc) [inline .github/workflows/build-firmware.yaml:197]: Please add code comments explaining what it does and how? i.e. both user documentation and implementation details?

**Review by rusefillc — CHANGES_REQUESTED** [review]

**Comment by rusefillc** [comment]
@chuckwagoncomputing I am sorry I do not get it :(

My understanding is that all master builds are the same. At the moment my manual release process is to download files from https://rusefi.com/build_server/ and to manually attach those to new GH release

I am wonder if/what is the smarter way to release but at the moment I am not aware of a better process. Is your fix assuming some smarter-than-what-I-do release process?

**Comment by chuckwagoncomputing** [comment]
Oh I'm the one stupid here. I remembered seeing branches in the branch list with "release" in the name, e.g. 2017.02.20_release_1.0.1

**Comment by rusefillc** [comment]
@chuckwagoncomputing yes while creating a release I think GHA demands a tag or branch name so I manually type the name of new branch to create.

Which is horrible since technically it's not atomic - branch is created while creating release and the artifacts are manually uploaded separately.

In my opinion with just monthly releases it's NOT worth to automate it unless someone knows a REALY cheap way to automate this.

For the specific task at hand can we have a manually invoked action which does same firmware like master but with a special environment variable which we would use to remove the randomness?

**Pull request closed without merging**